documentation/docs/practical-sessions/converting-synop-data-to-bufr.md

```diff
@@ -134,21 +137,20 @@ Click submit. The data will now be converted to BUFR and the result returned to
 ??? success "Click to reveal answer"
     Clicking the inspect button should bring up a dialog like that shown below.
 
-    <center><img alt="Results of the inspect button showing the basic station metadata, the station location and the observed properteis"
+    <center><img alt="Results of the inspect button showing the basic station metadata, the station location and the observed properties"
          src="../../assets/img/synop2bufr-ex2.png"></center>
 
-    This includes the station location shown on a map and basic metadata, as well as the observations, 
-    extracted from the BUFR message. As part of the transformation from FM-12 SYNOP to BUFR this information
-    is merged from the station metadata. The BUFR file can also be inspected by downloading the file
-    and validating using a tool as as the ECMWF ecCodes BUFR validator.
+    This includes the station location shown on a map and basic metadata, as well as the observations in the message.
+    
+    As part of the transformation from FM-12 SYNOP to BUFR, additional metadata was added to the BUFR file.
+    
+    The BUFR file can also be inspected by downloading the file and validating using a tool such as as the ECMWF ecCodes BUFR validator.
 
 As a final step navigate to the monitoring page from the left menu and confirm that the data have been published.
 
 <center><img alt="Image showing monitoring tab in on the left menu" src="../../assets/img/csv2bufr-monitoring.png"/></center>
```

Comment:
Image showing monitoring icon in the left navigation drawer*

*I don't know how we refer to this elsewhere.